spec/models/category_spec.rb

```diff
@@ -5,28 +5,29 @@
   it { should validate_presence_of(:name)}
 
   describe "recent videos" do 
+    before do 
+      @drama = Category.create(name: "Drama")
+    end 
+    let(:subject) { @drama.recent_videos }
+
     it "should show only 6 videos if more than 6 videos exists in a category" do 
-      thriller = Category.create(name: "thriller")
       7.times do 
-        video = Video.create(title:"24", description: "Fast thriller", category_id: thriller.id)
+        Fabricate(:video, category_id: @drama.id)
```

Comment:
`7.times` - Fabrication has this nifty method as well `Fabricate.times(7, :video, category: drama)` <-- note `drama` is defined via `let(:drama) { Fabricate(:category) }`